### PR TITLE
set yaml-cpp to download version 0.6.3

### PR DESCRIPTION
**Pull request by dkeeney**
This is the newest release of yaml-cpp.   The master which we were building from now has breaking errors.  The new release v0.6.3 seems to work ok.

Have not checked to see if this fixes the problem of building in debug mode on Linux.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v2.0.14
\ No newline at end of file
+v2.0.16
```
Comment (breznak) [inline VERSION]: OT: can we add the VERSION to gitignore? It;s autogenerated anyway

File: external/YamlCppLib.cmake [hunk 2]
```diff
@@ -21,8 +21,8 @@ if(EXISTS "${REPOSITORY_DIR}/build/ThirdParty/share/yaml-cpp.zip")
     set(URL "${REPOSITORY_DIR}/build/ThirdParty/share/yaml-cpp.zip")
 else()
     #set(URL https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.2.tar.gz)
-    # There seems to be something wrong with the 0.6.2 distribution.  Use the master.
-    set(URL https://github.com/jbeder/yaml-cpp/archive/master.zip)
+    # There seems to be something wrong with the 0.6.2 distribution.  Use the v0.6.3.
+    set(URL https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.3.zip)
```
Comment (breznak) [inline external/YamlCppLib.cmake]: great1

File: VERSION (hunk 1 above)
Comment (dkeeney) [reply VERSION]: sure, I can do that in this PR

**Comment by dkeeney** [comment]
I just checked.....this does NOT fix the problem with of the crash in debug on linux.

**Comment by dkeeney** [comment]
Oh, you deleted my branch.  I will not be able to use this one to set VERSION to gitignore.

**Pull request merged**